thunder/torch/__init__.py

```diff
@@ -239,10 +239,11 @@ def is_floating_point(a: TensorLike, /) -> bool:
 
 
 # Handles the size method
+@torchsymbol(torch.Tensor.size)
```

Comment:
I'm not certain, but I think 
```suggestion
@torchsymbol(torch.Tensor.size, is_method=True)
```
might help with the doc build failure.

Reply:
Apparently not. :(

Reply:
So while the above is a good amendment, it does not fix the type annotation problem, caused by

https://github.com/Lightning-AI/lightning-thunder/blob/32693249987182f2d691543b5298539c8d901f1f/docs/source/reference/torch/index.rst?plain=1#L14

(The error is the resaon why no symbols have been added there, but size was not a symbol, so while it would be nice to have a reference, let us not block on this here and remove the line above for now.)